File: Program Python/Vehicle.py
class Vehicle:
    def __init__(self, plat_nomer, merk, tahun_produksi, warna):
        self.plat_nomer = plat_nomer
        self.merk = merk
        self.tahun_produksi = tahun_produksi
        self.warna = warna

    def get_plat_nomer(self):
        return self.plat_nomer

    def get_merk(self):
        return self.merk

    def get_tahun_produksi(self):
        return self.tahun_produksi

    def get_warna(self):
        return self.warna

    def get_description(self):
        return "Tidak Ada Deskripsi"

File: Program Cpp/Vehicle.cpp
// Menggunakan pragma once agar header file hanya di-include satu kali
#pragma once

#include <iostream>
#include <string>
#include <vector>
// Memasukan library yang digunakan

using namespace std;

class Vehicle
{
	// Class Vehicle untuk parent dari Class Car dan Motorcycle

private:
	string plat_nomer;
	string merk;
	string tahun_produksi;
	string warna;
	// Atribute yang dibutuhkan

public:
	Vehicle(string plat_nomer, string merk, string tahun_produksi, string warna){
		this->plat_nomer = plat_nomer;
		this->merk = merk;
		this->tahun_produksi = tahun_produksi;
		this->warna = warna;
	}
	// Constructor

	string getPlatNomer(){
		return this->plat_nomer;
	}
	
	string getMerk(){
		return this->merk;
	}

	string getTahunProduksi(){
		return this->tahun_produksi;
	}

	string getWarna(){
		return this->warna;
	}

	string getDescription(){
		return "Tidak Ada Deskripsi";
	}

};